src/ontology/mondo-edit.obo

```diff
@@ -159781,8 +159787,8 @@ xref: MESH:C564133 {source="MONDO:equivalentTo"}
 xref: NCIT:C3116 {source="MONDO:equivalentTo"}
 xref: OMIM:147050 {source="MONDO:equivalentTo"}
 xref: UMLS:C1840253 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:327063"}
+is_a: MONDO:0000605 {source="PMID:10510320"} ! hypersensitivity reaction disease
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
-is_a: MONDO:0004980 ! atopic eczema
```

Comment:
this should be an excluded subclass of

Reply:
@nicolevasilevsky should this still be an excluded subclass? The PR was already merged.

Reply:
oops I didn't meant to merge it!
I'll fix this - I'll exclude the subclass